pysrc/i2t.py

```
@@ -310,6 +310,15 @@ def ml_ib_form_prepare(self, doc):
         """
         self._impl.ml_ib_form_prepare(doc)
 
+    def ub_classify_form(self, doc, img):
```

Ruff (0.12.2)

Missing return type annotation for private function `ub_classify_form`

(ANN202)